public_suffix_list.dat

```diff
@@ -13866,6 +13866,10 @@ pcloud.host
 // Submitted by Matthew Brown <mattbrown@nyc.mn>
 nyc.mn
 
+// Obl.ong : <https://obl.ong>
+// Submitted by Reese Armstrong <reese@obl.ong>
```

Comment:
Please use a non-personal e-mail.

Reply:
Not a personal email

Reply:
Would you rather I use team@obl.ong?

Reply:
Yes, just use something that doesn't refer to a single person who might leave the organization.